### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by samahjamie**
please check this draft, and try to put your ideas about the solution and problem. @techhana2014 
@Ibrahimnabeel9  kindly add the desktop responsive image.

### CODE REVIEW

**Review by yazan05 — COMMENTED** [review]
Make :"(( Project Name
Links
Problem 
etc .. )) a hyperlink 

Decide which way you will use to navigate between pages , to edit button design in User Journey >> **Next** or **Swipe**

**Review by Ibrahimnabeel9 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,58 +1,68 @@
...
 
+### **Project Name:** <span id='about'></span>
+COVIDtracker
 
-### **About COVIDtracker** <span id='about'></span>
+### **Links:** <span id='live'></span>
+* You can visit our live website [Here](https://)
 
-COVIDtracker is an application made to inform you about the number of people infected by Covid-19 virus in your country and fully aware users of all cases.
+* [Our figma design](https://)
```
Comment (techhana2014) [inline README.md:20]: where is figma link!

File: README.md [hunk 2]
```diff
@@ -1,58 +1,68 @@
...
 
-![](https://i.imgur.com/HRS8B2H.jpg)
+* Our API documentation: [1 2 3](https://)
```
Comment (techhana2014) [inline README.md:22]: and API links?

File: README.md [hunk 3]
```diff
@@ -1,58 +1,68 @@
...
 
 
-### **Features** <span id='features'></span>
-* As a user, you will be able to read the instructions to prevent the spread of the Covid-19 virus.
-* As a user, you will be able to be aware about the infected cases and deaths numbers in your country.
-* As a user, you will be able to skip instructions every time you access the page.
+### **Problem:** <span id='about'></span>
```
Comment (techhana2014) [inline README.md:25]: here we should write about problems

File: README.md [hunk 4]
```diff
@@ -1,58 +1,68 @@
...
 
+<!-- COVIDtracker is an application made to inform you about the number of people infected by Covid-19 virus in your country and fully aware users of all cases.
 
-### **User Journey**
-The user enters the app and presses continue button through the home page. When you click on the start button, it moves you to the next page where you can show the instructions of COVID-19, then you can skip or go to the next page that you can choose the country to know the number of active cases, recovered people and total deaths.
+![](https://i.imgur.com/HRS8B2H.jpg) -->
 
-![](https://i.imgur.com/zP1Ac70.jpg)
+### **Solution:** <span id='about'></span>
```
Comment (techhana2014) [inline README.md:31]: where is solutions samah

File: README.md [hunk 5]
```diff
@@ -1,58 +1,68 @@
...
+* [About Us](#about-us)
 
 
 
+### **Project Name:** <span id='about'></span>
+COVIDtracker
 
-### **About COVIDtracker** <span id='about'></span>
+### **Links:** <span id='live'></span>
+* You can visit our live website [Here](https://)
```
Comment (techhana2014) [inline README.md:18]: where is live link?

**Review by techhana2014 — CHANGES_REQUESTED** [review]
there is some changes in readme

**Pull request closed without merging**